xml/System.Linq/Enumerable.xml

```diff
@@ -12608,12 +12608,12 @@ Only unique elements are returned.
         <Parameter Name="second" Type="System.Collections.Generic.IEnumerable&lt;TSecond&gt;" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <typeparam name="TFirst">To be added.</typeparam>
-        <typeparam name="TSecond">To be added.</typeparam>
-        <param name="first">To be added.</param>
-        <param name="second">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TFirst">The type of the elements of the first input sequence.</typeparam>
+        <typeparam name="TSecond">The type of the elements of the second input sequence.</typeparam>
+        <param name="first">The first sequence to merge.</param>
+        <param name="second">The second sequence to merge.</param>
+        <summary>Applies a specified function to the corresponding elements of two sequences, producing a sequence of value types with elements from both sequences.</summary>
+        <returns>An <see cref="T:System.Collections.Generic.IEnumerable`1" /> that contains merged value types with first elements taken from first sequence and second elements taken from second sequence.</returns>
```

Comment:
Ideally we'd describe this without putting the type returned since it's auto-generated info and because it's generic, we can't really format this to render `IEnumerable<ValueTuple<TFirst,TSecond>>` instead of `IEnumerable<T>`

Perhaps

```suggestion
        <returns>A sequence that contains merged value types with elements taken from the first and second sequences, in that order.</returns>
```